patch-package
--- a/node_modules/table-dragger/dist/table-dragger.js
+++ b/node_modules/table-dragger/dist/table-dragger.js
@@ -3248,14 +3248,16 @@ return /******/ (function(modules) { // webpackBootstrap
 	    mousemove: 'pointermove'
 	  };
 	  var microsoft = {
-	    mouseup: 'MSPointerUp',
-	    mousedown: 'MSPointerDown',
-	    mousemove: 'MSPointerMove'
+	    mouseup: 'mouseup',
+	    mousedown: 'mousedown',
+	    mousemove: 'mousemove'
 	  };
 	  if (global.navigator.pointerEnabled) {
-	    crossvent[op](el, pointers[type], fn);
-	  } else if (global.navigator.msPointerEnabled) {
-	    crossvent[op](el, microsoft[type], fn);
+	    if (global.navigator.msPointerEnabled) {
+	      crossvent[op](el, microsoft[type], fn);
+	    } else {
+	      crossvent[op](el, pointers[type], fn);
+	    }
 	  } else {
 	    crossvent[op](el, touch[type], fn);
 	    crossvent[op](el, type, fn);
@@ -4220,6 +4222,12 @@ return /******/ (function(modules) { // webpackBootstrap
 	  mousemove: 'pointermove'
 	};
 
+	var microsoft = {
+	  mouseup: 'mouseup',
+	  mousedown: 'mousedown',
+	  mousemove: 'mousemove'
+	};
+
 	var getTouchyEvent = exports.getTouchyEvent = function getTouchyEvent() {
 	  var event = void 0;
 	  if (global.navigator.pointerEnabled) {
@@ -4249,7 +4257,11 @@ return /******/ (function(modules) { // webpackBootstrap
 
 	var touchy = exports.touchy = function touchy(el, op, type, fn) {
 	  if (global.navigator.pointerEnabled) {
-	    _crossvent2.default[op](el, pointers[type], fn);
+	    if (global.navigator.msPointerEnabled) {
+	    _crossvent2.default[op](el, microsoft[type], fn);
+	    } else {
+	      _crossvent2.default[op](el, pointers[type], fn);
+	    }
 	  } else {
 	    _crossvent2.default[op](el, touch[type], fn);
 	    _crossvent2.default[op](el, type, fn);
